tasks/new_e2e_tests.py

```diff
@@ -133,6 +133,12 @@ def run(
     )
 
     success = process_test_result(test_res, junit_tar, AgentFlavor.base, test_washer)
+
+    if running_in_ci():
+        # Do not print all the params, they could contain secrets needed only in the CI
+        targets_params = [f'--targets {t}' for t in targets]
+        print(f"To run this command locally, use: `inv -e new-e2e-tests.run {' '.join(targets_params)}`")
```

Comment:
I think we should the prefix `E2E_PIPELINE=<pipeline-id>` so the users will run the test with the same artifact as the one in the CI. I think we will also need to `aws-vault exec sso-agent-sandbox-account-admin --` to authenticate on AWS.
Something like:
```
E2E_PIPELINE_ID=<pipeline id> aws-vault exec sso-agent-sandbox-account-admin -- inv -e new-e2e-tests.run {' '.join(targets_params)}
```

Do you think we could print more of the params? For example for installer tests we need to specify more things to have the test running cf [here](https://github.com/DataDog/datadog-agent/blob/main/test/new-e2e/README.md?plain=1#L39)

Reply:
Yes sure!

Reply:
Addressed, let me know what you think

Reply:
Looks good like that, thanks!